riscv/mmu.h

```diff
@@ -187,6 +211,21 @@ class mmu_t
     })
   }
 
+  // for shadow stack amo swap
+  template<typename T, typename op>
+  T ss_amo(reg_t addr, op f) {
+    convert_load_traps_to_store_traps({
+      bool forced_virt = false;
+      bool hlvx = false;
+      bool lr = false;
+      bool ss_access = true;
+      store_slow_path(addr, sizeof(T), nullptr, {forced_virt, hlvx, lr, ss_access}, false, true);
```

Comment:
Do we need to do alignment check here? ssamoswap requires address to be naturally aligned to size.

Reply:
store_slow_path already check required_aligned parameter with true
Do we need to check outside ?

Reply:
Thanks. If `store_slow_path` does it then we don't need to add check outside.